search.py

```diff
@@ -493,7 +494,7 @@ class NPuzzle(Problem):
 
 
 
-    def __init__(self, initial= None, goal= None, size = 3, heuristic = 'manhattan_distance', shuffle =10):
+    def __init__(self, initial= None, goal= None, size = 3, heuristic = 'hamming', shuffle =10):
```

Comment:
Don't leave spaces around '='. For example, it should be `size=3` not `size = 3`.